content/articles/spring-webflux/index.md

```diff
@@ -126,7 +126,7 @@ public class ApplicationConfig {
     }
 }
 ```
- - The code snippet above sets the path to the `application.properties` file, it contains the project configurations.
+ - The code snippet above sets the path to the `application.properties` file. It contains the project configurations.
```

Comment:
Change *It* to *This file* because it is a bit vague as to what you're talking about there.